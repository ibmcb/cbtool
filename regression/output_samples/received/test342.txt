The following HOSTs reported management metrics:
Name                        |Age (seconds)     |Experiment id                  |Number of samples
host_simhosta0              |682               |REGRESSION_B                   |1                
host_simhostb0              |682               |REGRESSION_B                   |1                
host_simhostd0              |682               |REGRESSION_B                   |1                
host_simhostc0              |682               |REGRESSION_B                   |1                
host_simhostb1              |682               |REGRESSION_B                   |1                
host_simhosta1              |682               |REGRESSION_B                   |1                
host_simhostd1              |682               |REGRESSION_B                   |1                
host_simhostc1              |682               |REGRESSION_B                   |1                
host_simhostb2              |682               |REGRESSION_B                   |1                
host_simhostd2              |682               |REGRESSION_B                   |1                
host_simhosta2              |682               |REGRESSION_B                   |1                
host_simhostc2              |682               |REGRESSION_B                   |1                
host_simhostd3              |682               |REGRESSION_B                   |1                
host_simhostb3              |682               |REGRESSION_B                   |1                
host_simhosta3              |682               |REGRESSION_B                   |1                
host_simhostc3              |682               |REGRESSION_B                   |1                
host_simhosta4              |682               |REGRESSION_B                   |1                
host_simhostd4              |682               |REGRESSION_B                   |1                
host_simhostb4              |682               |REGRESSION_B                   |1                
host_simhostc4              |682               |REGRESSION_B                   |1                

The following HOSTs reported runtime (OS) metrics:
Name                        |Age (seconds)     |Experiment id                  |Number of samples

