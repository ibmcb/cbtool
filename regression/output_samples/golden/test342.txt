The following HOSTs reported management metrics:
Name                        |Age (seconds)     |Experiment id                  |Number of samples
host_simhostb0              |642               |REGRESSION_B                   |1                
host_simhosta0              |642               |REGRESSION_B                   |1                
host_simhostc0              |642               |REGRESSION_B                   |1                
host_simhostd0              |642               |REGRESSION_B                   |1                
host_simhostb1              |642               |REGRESSION_B                   |1                
host_simhosta1              |642               |REGRESSION_B                   |1                
host_simhostd1              |642               |REGRESSION_B                   |1                
host_simhostc1              |642               |REGRESSION_B                   |1                
host_simhostb2              |642               |REGRESSION_B                   |1                
host_simhosta2              |642               |REGRESSION_B                   |1                
host_simhostd2              |642               |REGRESSION_B                   |1                
host_simhostc2              |642               |REGRESSION_B                   |1                
host_simhostb3              |642               |REGRESSION_B                   |1                
host_simhosta3              |642               |REGRESSION_B                   |1                
host_simhostd3              |642               |REGRESSION_B                   |1                
host_simhostc3              |642               |REGRESSION_B                   |1                
host_simhostb4              |642               |REGRESSION_B                   |1                
host_simhosta4              |642               |REGRESSION_B                   |1                
host_simhostd4              |642               |REGRESSION_B                   |1                
host_simhostc4              |642               |REGRESSION_B                   |1                

The following HOSTs reported runtime (OS) metrics:
Name                        |Age (seconds)     |Experiment id                  |Number of samples

